index.html

```diff
@@ -86,34 +86,124 @@
     <script>
         var read_storage = true;
         var project_src;
-        var query = window.location.search.substring(1);
-        if (query.length > 0) {
-            var pair = query.split("=");
-            if (pair.length == 2 && pair[0] == "project") {
-                project_name = pair[1];
-                read_storage = false;
-                var client = new XMLHttpRequest();
-                client.open("GET", "pynode_projects/" + project_name + ".py");
-                client.onreadystatechange = function () {
-                    if (client.readyState == 4) {
-                        project_src = client.responseText;
+
+        const searchParams = new URLSearchParams(window.location.search);
+
+        if (searchParams.get("project")) {
+            project_name = pair[1];
+            read_storage = false;
+            var client = new XMLHttpRequest();

```

Comment:
nit: let's use `const` wherever possible

Reply:
I've changed the `vars` into `lets`